### DISCUSSION TITLE
✨✨✨ INTERTESTING FORKS ✨✨✨

### DISCUSSION THREAD

**Discussion by egzumer** [egzumer/uv-k5-firmware-custom#485] [Show and tell]
There is a work done by others on forks of this repository. I encourage you to take a look at those too.
You can use below links to see comparison between forks.<br>
https://gitpop2.vercel.app/egzumer/uv-k5-firmware-custom<br>
https://useful-forks.github.io/?repo=egzumer/uv-k5-firmware-custom

Disclaimer: Merging other forks forces me to analyze the code, hunt bugs, make changes to chirp driver and so on. That is a lot of work and I'm not keen on doing that at this point, but I will do anything to promote work of others. If you added some functionalities in your fork and want to share it, please post a link and description below to help others find your work.

Tip: you can use web flashing tool  `https://egzumer.github.io/uvtools/?firmwareURL=` to post direct link to your firmware bin by appending a link to your binary at the end, after equal sign.

Lets keep this thread clean, no discussions here.

**Comment by AubsUK** [comment]
I've just finished the second minor version of firmware and CHIRP driver over on my repo - https://github.com/AubsUK/uv-k5-firmware-custom

The current main changes to Egzumer's firmware are:

* The two ScanLists have been removed and replaced with 10 ScanLists (0-9)
* When scanning, the ScanLists now show along the top of the screen
* Toggling ScanLists on/off is as simple as pressing the number
* Memory channels can now be permanently excluded (Lockout/Skip) from scanning
* Memory channels can now be temporarily excluded while scanning (press `*` will lock out the channel until a power cycle)
* Scanning can be set to automatically start when the radio is turned on
* All the changes can be managed through CHIRP

You WILL need a custom CHIRP driver to use CHIRP.  All instructions provided.

Open a discussion there if you have any questions or suggestions.

**Comment by nikant** [comment]


**Reply by gulbalee** [reply]
Hi Nikos! 
Your fork is wonderful however the source code is not correct, it is that of @egzumer with modifications made by @kamilsss655  and the same goes for your other firmware available at https://github.com/nikantgr/egzumer-uv-k5-firmware-custom-nikant/releases/tag/nk0.0.7 
Please share your source code on github as this is under GNU GPL v3 license and the source code should be shared on request.

Thank you so much for the great work and for contributing to the open source community. Looking forward to your code becoming available.

Regards!

**Comment by nikant** [comment]
and another remix directly from egzumer's 

https://github.com/nikantgr/egzumer-uv-k5-firmware-custom-nikant/releases

https://www.youtube.com/watch?v=lJBkC0Dh5mQ
